make/images.mk

```diff
@@ -443,6 +443,23 @@ rhel86-fips-offline:
 rhel86-offline-nvidia:
 	$(MAKE) aws-rhel-8.6_offline-nvidia
 
+# Rocky 9.1
+.PHONY: rocky91
+rocky91:
+	$(MAKE) build-aws-rocky-9.1
+
+.PHONY: rocky91-offline
+rocky91-offline:
+	$(MAKE) aws-rocky-9.1_offline
+
+.PHONY: rocky91-fips-offline
```

Comment:
Did we build FIPS offline packages? I do't think RockyLinux supports the correct `140-2` FIPS version only https://rockylinux.org/news/certifications-fips-2022-06-11/ `140-3`.

Reply:
FIPS offline packages for rocky. [Link](https://teamcity.mesosphere.io/buildConfiguration/MesosphereOnly_ClosedSource_DkpReleases_OsPackagesBundles_Rocky9fips/3789491?buildTab=log&expandAll=true&focusLine=2019)
I have still not run any FIPS validation tests